fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/segmentedcontrols/Pill.kt

```diff
@@ -164,7 +167,10 @@ fun PillButton(
             .then(if (interactionSource.collectIsFocusedAsState().value || interactionSource.collectIsHoveredAsState().value) focusedBorderModifier else Modifier)
             .padding(vertical = token.verticalPadding(pillButtonInfo))
             .semantics(true) {
-                contentDescription = if (pillMetaData.enabled) "${pillMetaData.text} $selectedString" else "${pillMetaData.text} $enabledString"
+                contentDescription =
+                    (if (pillMetaData.enabled) "${pillMetaData.text} $selectedString"
```

Comment:
Lets update this string in pillbar instead and not touch pillbutton since the issue is in pillbar

Reply:
overriding this part from pillbar

Reply:
overriding this part from pillbar